cloud: add the reminder message of charging policy for backups when customers delete dedicated clusters (#17581)
pingcap/docs: tidb-cloud/delete-tidb-cluster.md

    > Alternatively, you can also click the name of the target cluster to go to its overview page, and then click **...** in the upper-right corner.

3. Click **Delete** in the drop-down menu.

> 4. In the cluster deleting window, the prompt message will inform you of the number of backups and the charging policy for backups if you have at least one manual or automatic backups. If you click **Continue** or if no backups are present, you will be prompted to enter `<organization name>/<project name>/<cluster name>` to confirm the deletion.

Reviewer: ```suggestion
4. In the cluster deleting window, confirm the deletion.

    - If you have at least one manual or automatic backup, you can see the number of backups and the charging policy for backups. Click **Continue** and enter `<organization name>/<project name>/<cluster name>`.
    - If you do not have any backups, just enter `<organization name>/<project name>/<cluster name>`.
```

    If you want to restore the cluster sometime in the future, make sure that you have a backup of the cluster. Otherwise, you cannot restore it anymore. For more information about how to back up TiDB Dedicated clusters, see [Back Up and Restore TiDB Dedicated Data](/tidb-cloud/backup-and-restore.md).

    > **Note:**
    >
    > [TiDB Serverless clusters](/tidb-cloud/select-cluster-tier.md#tidb-serverless) only support [in-place restoring from backups](/tidb-cloud/backup-and-restore-serverless.md#restore) and do not support restoring data after the deletion. If you want to delete a TiDB Serverless cluster and restore its data in the future, you can use [Dumpling](https://docs.pingcap.com/tidb/stable/dumpling-overview) to export your data as a backup.

5. Click **I understand, delete it**.

 Once a backed up TiDB Dedicated cluster is deleted, the existing backup files of the cluster are moved to the recycle bin.

> - Automatic backups will expire and be automatically deleted once the retention period has passed. The default retention period is 7 days if you don't modify it. 
> - Manual backups will be kept in the Recycle Bin until manually deleted.

Reviewer: ```suggestion
- Automatic backups will stop and be automatically deleted once the retention period expires. The default retention period is 7 days if you don't modify it.
- Manual backups will be kept in the Recycle Bin until manually deleted.
```

Author: Thanks for comment, but I think stop is not accurate for automatic backup. It's not stop backup but backup files will be expired when the retention time arrived.

Reviewer: Got the meaning. Then let's keep "expire".

    If you want to restore the cluster sometime in the future, make sure that you have a backup of the cluster. Otherwise, you cannot restore it anymore. For more information about how to back up TiDB Dedicated clusters, see [Back Up and Restore TiDB Dedicated Data](/tidb-cloud/backup-and-restore.md).

    > **Note:**
    >
    > [TiDB Serverless clusters](/tidb-cloud/select-cluster-tier.md#tidb-serverless) only support [in-place restoring from backups](/tidb-cloud/backup-and-restore-serverless.md#restore) and do not support restoring data after the deletion. If you want to delete a TiDB Serverless cluster and restore its data in the future, you can use [Dumpling](https://docs.pingcap.com/tidb/stable/dumpling-overview) to export your data as a backup.

5. Click **I understand, delete it**.

 Once a backed up TiDB Dedicated cluster is deleted, the existing backup files of the cluster are moved to the recycle bin.

- Automatic backups will expire and be automatically deleted once the retention period has passed. The default retention period is 7 days if you don't modify it. 
- Manual backups will be kept in the Recycle Bin until manually deleted.

  > **Note:**
  >

>   > Please be aware that backups will continue to incur charges until deleted.

Reviewer 2: ```suggestion
  > Backups will continue to incur charges until deleted.
```

Reviewer: The original sentence is exactly the one that is displayed on the TiDB Cloud console page. So I think it's ok to keep it as it is?
